tests/e2e/scenarios/test_skills.py

```
@@ -1,78 +1,76 @@
-    search_input = page.locator(SEL["skill_search_input"])
-    await search_input.fill("markdown")
-    await search_input.press("Enter")
+    """Install a skill from search, then remove it from the installed list."""
+    search = await _open_skills(page)
+    await search.fill("markdown")
 
+    results = page.locator(SEL["skill_search_result"])
     try:
-        results = page.locator(SEL["skill_search_result"])
         await results.first.wait_for(state="visible", timeout=20000)
     except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)